lib/embulk/input/twitter_ads/card.rb

```diff
@@ -0,0 +1,63 @@
+require 'json'
+require_relative 'api'
+require_relative 'util'
+
+module Embulk
+  module Input
+    module TwitterAds
+      class Card
```

Comment:
![memo-badge](https://img.shields.io/badge/review-memo-lightgrey)
👍 
we need refactoring to create a class for each resource.